app/furniture/marketplace/product.rb

```diff
@@ -41,6 +41,9 @@ class Product < Record
 
     before_commit :standardize_attachment_name
 
+    scope :with_group_tag, -> { joins(:tags).merge(Tag.group_tag) }
+    scope :without_group_tag, -> { where.not(id: with_group_tag) }
```

Comment:
Cool composition of scopes to replace the raw query!

Reply:
Ha, I definitely overthought the original query, because this one that AR produces with a subquery is concise: 

```sql
select
	"marketplace_products".*
from
	"marketplace_products"
where
	"marketplace_products"."id" not IN(
		select
			"marketplace_products"."id" from "marketplace_products"
			inner join "marketplace_product_tags" on "marketplace_product_tags"."product_id" = "marketplace_products"."id"
			inner join "marketplace_tags" on "marketplace_tags"."id" = "marketplace_product_tags"."tag_id"
		where
			"marketplace_tags"."is_group" = true)
limit 11;
```